src/art.js

```diff
@@ -579,7 +579,14 @@
     "art_icon": "./art/yellowbeard_2/icon.png",
     "author_name": "Mayank Goel",
     "author_github_url": "https://github.com/yellowberard/"
-  }
+  },
```

Comment:
Please revert this file back. You are not allowed to edit the `src/art.js` file. By fixing this, you should also remove the merge conflict that is present in this PR.
![image](https://user-images.githubusercontent.com/48055438/194716748-ce06bada-9978-44d8-8a61-def67200f8ab.png)